pkg/cmd/app_test.go

```diff
@@ -205,11 +196,11 @@ func requireSetenv(t *testing.T, key, value string) func() {
 	}
 }
 
-// newRootCommand initializes a command with buffers for stdout and stderr.
-func newRootCommand(o *globals.GlobalOpts) (c *cobra.Command, stdout, stderr *bytes.Buffer) {
+// newApp initializes a command with buffers for stdout and stderr.
+func newApp(o *globals.GlobalOpts) (c *cobra.Command, stdout, stderr *bytes.Buffer) {
```

Comment:
chose name app as the root command represents the application. also, in urlfave it is called app